### PR TITLE
Merging long haul tests to LiSA branch

### PR DESCRIPTION
**Pull request by shruti25ratnam**

### CODE REVIEW

**Review by dsrivastavv — APPROVED** [review]

File: microsoft/testsuites/acc/longhaul.py [hunk 1]
```diff
@@ -43,28 +44,33 @@
 )
 class ACCPerformance(TestSuite):
     TIME_OUT = 15000
-    IOSTAT_OUTPUT_PATH = "/home/lisatest/log.txt"
+    MEASUREMENT_COUNT = 300
+    MEASUREMENT_INTERVAL = 40
+    IOSTAT_OUTPUT_PATH = "/home/lisatest/log_disk.txt"
```
Comment (LiliDeng) [inline microsoft/testsuites/acc/longhaul.py:49]: do not hardcode username

File: microsoft/testsuites/acc/longhaul.py [hunk 2]
```diff
@@ -43,28 +44,33 @@
...
     def monitor_disk_throughput(self, node: Node, environment: Environment) -> None:
         # wait for 10 minutes
-        timeout = 60
+        timeout = self.MEASUREMENT_COUNT * self.MEASUREMENT_INTERVAL
         try:
             result = node.execute("apt-get install -y sysstat", sudo=True, shell=True)
         except:
             print("Unable to install sysstat")
-        iostat_command = f"iostat -p sda1 -dx 5 2 > {self.IOSTAT_OUTPUT_PATH}" 
+        iostat_command = f"iostat -p sda1 -dx {self.MEASUREMENT_INTERVAL} {self.MEASUREMENT_COUNT} > {self.IOSTAT_OUTPUT_PATH}"
```
Comment (LiliDeng) [inline microsoft/testsuites/acc/longhaul.py:65]: not hard code the disk name, seems run testing against os disk?

File: microsoft/testsuites/acc/longhaul.py [hunk 3]
```diff
@@ -79,6 +85,96 @@ def monitor_disk_throughput(self, node: Node, environment: Environment) -> None:
         if timeout < timer.elapsed():
             print(f"timeout {timeout} greater than elapsed {timer.elapsed()}")
 
+    def monitor_memory_throughput(self, node: Node, environment: Environment) -> None:
+        # wait for 10 minutes
+        timeout = self.MEASUREMENT_COUNT * self.MEASUREMENT_INTERVAL
+        try:
+            result = node.execute("apt-get install -y sysstat", sudo=True, shell=True)
```
Comment (LiliDeng) [inline microsoft/testsuites/acc/longhaul.py:92]: use `self.node.os.install_packages()`

File: microsoft/testsuites/acc/longhaul.py [hunk 4]
```diff
@@ -79,6 +85,96 @@ def monitor_disk_throughput(self, node: Node, environment: Environment) -> None:
...
+        except:
+            print("Unable to install sysstat")
```
Comment (LiliDeng) [inline microsoft/testsuites/acc/longhaul.py:94]: use log.debug

**Comment by LiliDeng** [comment]
feel free to reopen it when the code is ready for review, thanks.

**Comment by LiliDeng** [comment]
feel free to reopen it when the code is ready for review, thanks.

**Pull request closed without merging**